app/models/answer.rb

```diff
@@ -1,2 +1,11 @@
 class Answer < ActiveRecord::Base
+  validates :body, length: {minimum: 10, maximum: 3000}
+  validates :user_id, presence: true
```

Comment:
`validates :user` would be better, because it will actually ensure that there is a record for the given id.